Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -328,6 +341,12 @@ class TextureCacheBase
 
   void SetBackupConfig(const VideoConfig& config);
 
+  RcTcacheEntry CreateTextureEntry(const TextureCreationInfo& creation_info,
+                                   const TextureInfo& texture_info,
+                                   const int safety_color_sample_size,
```

Comment:
Seems like the const can be removed from this and `custom_arbitrary_mipmaps` but be left in the definition.

Aside from that LGTM